pom.xml

```diff
@@ -141,18 +141,21 @@
 
     <build>
         <plugins>
-            <plugin>
-                <groupId>io.quarkus</groupId>
-                <artifactId>quarkus-maven-plugin</artifactId>
-                <version>${quarkus-plugin.version}</version>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>build</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
+                <plugin>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-maven-plugin</artifactId>
+                    <version>${quarkus-plugin.version}</version>
+                    <extensions>true</extensions>
+                    <executions>
+                        <execution>
+                            <goals>
+                                <goal>build</goal>
+                                <goal>generate-code</goal>
```

Comment:
Why do we need these 2 new goals?

Reply:
https://github.com/quarkusio/quarkus/wiki/Migration-Guide-1.12#quarkus-maven-plugin

Reply:
Should we also add `<goal>generate-code-tests</goal>` ?

Reply:
The `generate-code-tests` goal has been also added. Line 154